lib/bundler.js

```diff
@@ -8,47 +8,51 @@ var cloneDeep = require('lodash').cloneDeep;
  * @param {Object} instructions Boot instructions.
  * @param {Object} bundler A browserify object created by `browserify()`.
  */
-
-module.exports = function addInstructionsToBrowserify(instructions, bundler) {
-  bundleModelScripts(instructions, bundler);
-  bundleMixinScripts(instructions, bundler);
-  bundleComponentScripts(instructions, bundler);
-  bundleOtherScripts(instructions, bundler);
-  bundleInstructions(instructions, bundler);
+module.exports = function addInstructionsToBrowserify(context, bundler) {
+  // bundlePluginScripts(context, bundler);
```

Comment:
I was trying to do so but ran into some require resolution issues. Can you help?